cobigen-templates/templates-devon4j/src/main/templates/crud_java_server_app/templates/java/${variables.rootPackage}/${variables.component}/common/api/${variables.entityName}.java.ftl

```diff
@@ -3,7 +3,13 @@ package ${variables.rootPackage}.${variables.component}.common.api;
 
 import ${variables.rootPackage}.general.common.api.ApplicationEntity;
 
-public interface ${variables.entityName} extends ApplicationEntity {
+<#assign compositeIdTypeVar = JavaUtil.getReturnTypeOfMethodAnnotatedWith(classObject,"javax.persistence.EmbeddedId")>
+<#if compositeIdTypeVar!="null"> 
+import ${variables.rootPackage}.general.common.api.ApplicationComposedKeyEntity;
```

Comment:
this will not exist anymore, and needed to be defined by the user manually, so we can remove it here